src/plugins/expression/expression-buttons.tsx

```diff
@@ -1,16 +1,62 @@
 import React from "react";
 import { Tooltip } from "react-tippy";
-// icon assets are the ones developed for the table tile
+import classNames from "classnames";
 import DeleteSelectionIcon from "../../assets/icons/delete/delete-selection-icon.svg";
 import { useTooltipOptions } from "../../hooks/use-tooltip-options";
+import { getCommand } from "./expression-tile-utils";
+import { MathfieldElement } from "mathlive";
+import { expressionButtonsList } from "./expression-types";
+import MixedFractionIcon from "./assets/mixed-fraction-icon.svg";
+import DivisionSymbolIcon from "./assets/division-symbol-icon.svg";
 
 import "./expression-toolbar.scss";
 
 interface IconButtonProps {
   className?: string;
-  onClick: (evt: React.MouseEvent) => void;
+  onClick?: (evt: React.MouseEvent | string) => void;
+  buttonName?: string;
 }
 
+interface AddMathTextButtonProps {
+  buttonName?: string;
+  enabled?: boolean;
+  mf: React.RefObject<MathfieldElement> | undefined;
+}
+
+const iconSvgs = {
+  "mixedFraction": <MixedFractionIcon />,
+  "mixedFractionSmart": <MixedFractionIcon />,
+  "divisionSymbol": <DivisionSymbolIcon />
+};
```

Comment:
Did you try adding an `icon` property to AddMathTextButton?
Then you could put the these SVG react components in `expression-types.ts`. You would have to rename it to `expression-types.tsx` so you could include JSX in the file.

Reply:
Oooooh - it was the file extension!  I had tried an icon property before to no avail. It works now.